### PR TITLE
added Suilend Cosmos Osmosis Sui  Axelar TradePort Jumper  PStake  Sp…

### PR DESCRIPTION
**Pull request by goodylili**
added Suilend Cosmos Osmosis Sui Axelar TradePort Jumper PStake SpringSui tradeport and others

### CODE REVIEW

**Comment by goodylili** [comment]
Gm @suhailkakar 

Everything good here?

**Comment by suhailkakar** [comment]
thank you, @goodylili 

there is a lot of eli5 here, so i'm just gonna take some time to review them and will merge it asap :)

**Comment by goodylili** [comment]
alright! I'm looking forward to that

**Comment by goodylili** [comment]
Hi @suhailkakar 

all good here?

**Comment by suhailkakar** [comment]
yep! merging it!
